app/renderer/js/utils/link-util.ts

```diff
@@ -36,7 +36,7 @@ export async function openBrowser(url: URL): Promise<void> {
     </body>
 </html>
 `);
-		shell.openItem(file);
+		await shell.openPath(file);
```

Comment:
@manavmehta  Can we also utilize the error message in the API design https://github.com/electron/governance/blob/master/wg-api/spec-documents/shell-openitem.md
in case an error occurs ? Just to be safe in this case.

Reply:
@vsvipul I have added a logger instance for the same. Do you have something else in mind?